Tools/Scripts/libraries/webkitbugspy/webkitbugspy/bugzilla.py

```diff
@@ -35,6 +35,7 @@
 
 requests = webkitcorepy.CallByNeed(lambda: __import__('requests'))
 
+requests.packages.urllib3.util.connection.HAS_IPV6 = False
```

Comment:
This isn't just scoped to `bugzilla`, so we probably need it at a higher level.

I think `webkitpy`'s `__init__.py` is the right place. That file has to be imported by anything that uses `requests` in WebKit, which seems like the right level to apply this logic at.

Reply:
I'm not objecting to landing a workaround, but we need to fix the root fix too (i.e. I'd expect to see a `# FIXME: Remove this when rdar://123456 is fixed`).